src/components/mobile/barcodeScanner.tsx

```diff
@@ -29,42 +35,86 @@ export class BarcodeScanner extends React.PureComponent<
   handleError(err: any) {
     console.error(err);
   }
-
-  render() {
-    const constraints = {
-      height: 720,
-      width: 1280,
-      facingMode: "user",
-      //   facingMode: { exact: "environment" }
+  constraints = {
+    height: 720,
+    width: 1280,
```

Comment:
These are hardcoded in bc I think they basically determine the image quality and not the actual size of the component as it renders? It was weird trying to get the camera to be the right size so I'm not entirely positive how this works but that seems to be it